draft-packages/form/KaizenDraft/Form/Primitives/Input/Input.module.scss

```diff
@@ -95,17 +103,28 @@ $input-with-icon-padding: calc(#{$input-icon-size} + calc(#{$ca-grid} * 0.75));
   }
 
   .input {
-    @include ca-padding(
-      $start: $input-with-icon-padding,
-      $end: $input-base-padding-horizontal
-    );
+    padding-right: $input-base-padding-horizontal;
```

Comment:
Just wondering, is this something where we should use `padding-inline-start/end` instead now that we don't have to support IE11 anymore?